README.md

```diff
@@ -1,41 +1,80 @@
+## Installation locally
 
-<!-- TODO: Add full-sized test dataset and amend the paragraph below if applicable -->
+You'll need to have [cookiecutter](https://cookiecutter.readthedocs.io/) installed.
 
-## Pipeline summary
+```{bash}
+pip install cookiecutter
+```
 
-<!-- TODO: Fill in short bullet-pointed list of the default steps in the pipeline -->
+## Usage
 
-## Quick start
+Run `cookiecutter gh:Arcadia-Science/nextflow-template` and then answer the prompts. Here's an example run:
 
-<!-- TODO: Fill in short bullet-pointed list of the default steps to get the pipeline up and running -->
+```{bash}
+$ cookiecutter gh:simonw/click-app
+```
 
-## Full documentation
+It is strongly recommended to accept the suggested value for "hyphenated" and "underscored" by hitting enter on those prompts.
 
-<!-- TODO: Fill in this section with how to fully use the pipeline, what the inputs are and what the outputs look like. If this section ends up being super long, feel free to create a new docs/ directory and add details there. -->
+Having created the new structure from the template, here's how to start working on the tool.
 
-## Contributions and support
+If your tool is called `my-new-tool`, you can start working on it like so:
 
-<!-- TODO: Add CONTRIBUTING.MD that is specific to Arcadia Science -->
+    cd my-new-tool
+    # Create and activate a virtual environment:
+    python3 -m venv venv
+    source venv/bin/activate
+    # Install dependencies so you can edit the project:
+    pip install -e '.[test]'
+    # With zsh you have to run this again for some reason:
+    source venv/bin/activate
+    # Confirm your tool can be run from the command-line
+    my-new-tool --version
 
-## Citations
+You should see the following:
 
-<!-- TODO: Add bibliography of tools and data used in your pipeline -->
+    my-new-tool, version 0.1
 
-An extensive list of references for the tools used by the pipeline can be found in the [`CITATIONS.md`](CITATIONS.md) file.
+You can run the default test for your tool like so:
+
+    pytest
+
+This will execute the test in `tests/test_my_new_tool.py`.
+
+Now you can open the `my_new_tool/cli.py` file and start adding Click [commands and groups](https://click.palletsprojects.com/en/7.x/commands/).
+
+## Creating a Git repository for your tool
+
+You can initialize a Git repository for your tool like this:
+
+```{bash}
+cd my-new-tool
+git init
+git add .
+git commit -m "Initial structure from template"
+# Rename the 'master' branch to 'main':
+git branch -m master main
+```
+
+## Publishing your tool to GitHub
+
+Use https://github.com/new to create a new GitHub repository sharing the same name as your tool, which should be something like `my-new-tool`.
+
+Push your `main` branch to GitHub like this:
+
+```{bash}
+git remote add origin git@github.com:YOURNAME/my-new-tool.git
+git push -u origin main
+```
```

Comment:
should we make this arcadia specific, or do you envision people outside of the org using this as well so keeping it general is good? if keeping it general is good, should we add a comment at least that for Arcadians the pipelines should go in Arcadia-Science

Reply:
Doesn't really matter, I expect Arcadia Science peeps to use this repo: https://github.com/Arcadia-Science/nextflow-template-repository